README.md

```diff
@@ -78,6 +78,9 @@ You'd have in your `app/views/foo/index.html.haml`:
 
 ```haml
 = table_for @foos [header_visible: (true|false), body_visible: (true|false), footer_visible: (true|false), downloadable: (:xlsx|:csv)] do
+  - row do |attributes,foo|
```

Comment:
mh, we are already adding attributes through the block - based on the object that is being rendered at the moment - so passing other attributes through parameters to `row` looks duplicate to me.

```ruby
- row do |attributes, foo|
  attributes['class'] = 'frobber'